bundles/org.openhab.binding.senechome/src/main/resources/OH-INF/i18n/senechome.properties

```diff
@@ -84,6 +85,7 @@ channel-type.senechome.powerLimitationState.label = Limitation State
 channel-type.senechome.powerMpp1.label = Power MPP1
 channel-type.senechome.powerMpp2.label = Power MPP2
 channel-type.senechome.powerMpp3.label = Power MPP3
+channel-type.senechome.saveChargeMode.label = Save Charge Mode
```

Comment:
These are old valuesw, don't forget to run the i18n tool: `mvn i18n:generate-default-translations`